site/content/tutorial/02-reactivity/04-updating-arrays-and-objects/text.md

```diff
@@ -13,15 +15,15 @@ function addNumber() {
 }
 ```
 
-But there's a more idiomatic solution:
+You can also get a similar result using the ES6 spread syntax:
```

Comment:
for some reason "similar" bugs me here as it's not a "similar" result but rather is the "same" result
```suggestion
You could also write this more concisely using the ES6 spread syntax:
```

Reply:
Well, there's the "creating new array" vs "using the same array" thing but I guess that's just me being pedantic. Sure.